std/range/package.d

```diff
@@ -5217,54 +5217,53 @@ auto sequence(alias fun, State...)(State args)
 
 // iota
 /**
-   Construct a range of values that span the given starting and stopping
-   values.
 
-   Params:
-   begin = The starting value.
-   end = The value that serves as the stopping criterion. This value is not
-        included in the range.
-   step = The value to add to the current value at each iteration.
+Creates a range of values that span the given starting and stopping values.
```

Comment:
This mixes whitespace changes and non-whitespace changes. Please never do this.

Even with ?w=1 it's not completely clean because some text has been rewrapped.